tutorials/misc/running_code_in_the_editor.rst

```diff
@@ -0,0 +1,93 @@
+.. _doc_running_code_in_the_editor:
+
+Running code in the editor
+==========================
+
+What is ``tool``?
+---------------
+
+``tool`` is a powerful line of code that, when added at the top of your script, makes it execute in the editor. You can also decide which parts of the script execute in the editor, which in game, and which in both.
+
+You can use it for doing many things, but it is mostly useful in level design for visually presenting things that are hard to predict ourselves. Here are some use cases:
+
+- If you have a cannon that shoots cannonballs affected by physics (gravity), you can draw cannonball's trajectory in the editor, making level design a lot easier.
+- If you have jumppads with varying jump heights, you can draw the maximum jump height a player would reach if it jumped on one, also making level design easier.
+- If your player doesn't use a sprite, but draws itself using code, you can make that drawing code execute in the editor to see your player.
+
+How to use it
+-------------
+
+To turn a script into a tool, add ``tool`` keyword at the top of your code, right under the ``extends ...`` line.

```

Comment:
I'd like to change this to *at the top of your code*, full stop, and rather change these new examples to have it at the very top. As far as I can see, all of the (admittedly few) examples thus far place the `tool` keyword in the very first line.

Reply:
That makes more sense, I will make all suggested changes.